projects/objects/road/protos/Road.proto

```diff
@@ -21,25 +21,25 @@ PROTO Road [
   field        SFFloat              speedLimit                -1.0                    # Optionally defines the speed limit. The recommended unit is meter per seconds.
   field        MFNode{RoadLine {}}  lines                     [ RoadLine { } ]        # Defines the property of each line separating two lanes.
   field        SFFloat              roadBorderHeight          0.15                    # Defines the height of the sidewalk.
-  field        MFFloat              roadBorderWidth           [ 0.8 ]                 # Defines the width of the sidewalk associated to each way-point (if there are less values than way-points, the last value is used for the last remaining way-points).
-  field        SFBool               road                      TRUE                    # Defines the width of the sidewalk associated to each way-point (if there are less values than way-points, the last value is used for the last remaining way-points).
+  field        MFFloat              roadBorderWidth           [ 0.8 ]                 # Defines the width of the sidewalk. associated to each way-point (if there are less values than way-points, the last value is used for the last remaining way-points).
+  field        SFBool               road                      TRUE                    # Defines the width of the sidewalk. associated to each way-point (if there are less values than way-points, the last value is used for the last remaining way-points).
```

Comment:
Why did you add a '.' in the middle of the sentence?

It seems that it was already correct before:
```suggestion
  field        SFBool               road                      TRUE                    # Defines the width of the sidewalk associated to each way-point (if there are less values than way-points, the last value is used for the last remaining way-points).
```